contracts/core/extensions/CoreIssuanceOrder.sol

```diff
@@ -36,17 +36,131 @@ contract CoreIssuanceOrder is
 {
     using SafeMath for uint256;
 
+    string constant INVALID_SIGNATURE = "Invalid order signature.";
+
+    struct IssuanceOrder {
+        address setAddress;
+        uint256 quantity;
+        address makerAddress;
+        address makerToken;
+        uint256 makerTokenAmount;
+        uint256 expiration;
+        address relayerToken;
+        uint256 relayerTokenAmount;
+        uint256 salt;
+        bytes32 orderHash;
+    }
+
```

Comment:
Add some descriptions including what’s at each index of the array params?

Reply:
Yeah needed to add javadocs and comments